CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 - [[PR437]](https://github.com/lanl/singularity-eos/pull/437) Fix segfault on HIP, clean up warnings, add strict sanitizer test
 
 ### Changed (changing behavior/API/variables/...)
+- [[PR475]](https://github.com/lanl/singularity-eos/pull/475) Shrink the default variant
```

Comment:
Can you add a note about the stiff gas specifically since I think we have that integrated in EAP?

Reply:
Done.